docs/installation/prerequisites-installation.md

```diff
@@ -1,18 +1,19 @@
-# Installation on local PC
-
 [Back](../README.md)
 
-## Prerequisites for Development Workplace
+## Prerequisites Installation
+
+In the description below, the following prerequisites are installed:
 
 * github account to login to projectbuilder
 * Java 14 or later
 * Maven 3
-* NodeJs with NPM - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* NodeJs with NPM (version 8.3.2) - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* docker (Version 20.10.8 or higher) and docker compose (version 1.29.2)
+* hint: if not using docker-compose scripts, mariaDB 10.6 is needed
 
 **Recommended:**
 
 * linux operating system
-* docker and docker compose
 * Visual Studio Code
```

Comment:
Or IntelliJ? I do not know if VSCode should be here....

Reply:
So wie oben - würde ich eher als ein Vorschlag nehmen. Du hast ja keine VSCode spezifischen Einstellungen in dem Projekt

Reply:
> So wie oben - würde ich eher als ein Vorschlag nehmen. Du hast ja keine VSCode spezifischen Einstellungen in dem Projekt

Das empfehle ich noch viel mehr...vs code hat eine recht hohe Bekanntheit